extensions/console-notification.crd.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolenotifications.console.openshift.io
+  annotations:
+    displayName: ConsoleNotification
+    description: Extension for configuring openshift web console notifications.
+spec:
+  group: console.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consolenotifications
+    singular: consolenotification
+    kind: ConsoleNotification
+    listKind: ConsoleNotificationList
+  additionalPrinterColumns:
```

Comment:
Would add the message and maybe the link as well, so user can see it though the CLI

Reply:
Good thought.  One possible down side is long messages make a mess of the table layout.  Thoughts?

![Screen Shot 2019-03-19 at 11 34 47 AM](https://user-images.githubusercontent.com/895728/54619519-1bb6c700-4a3b-11e9-8dfe-d62a5ee82006.png)

Reply:
@juanvallejo halp :) kidding.

Reply:
@rhamilto yeah I see your point but still think that it's better to see msgs in the cli then having almost empty output.